NuRadioReco/modules/LOFAR/planeWaveDirectionFitter_LOFAR.py

```diff
@@ -17,6 +17,56 @@
 from NuRadioReco.modules.LOFAR.beamforming_utilities import geometric_delay_far_field, lightspeed
 
 
+def average_direction(event, detector, mode='normal'):
+    """
+    Calculate the average direction for an event based on the plane wave directions of the individual stations.
+
+    Parameters
+    ----------
+    event : Event object
+        The event for which to calculate the average direction.
+    detector : Detector object
+        The detector for which to calculate the average direction.
+    mode : str, default='normal'
+        The mode to use for the calculation. Can be 'normal' (just raw mean) or 'weighted'
+        (with number of good antennas as weight per station).
+
+    Returns
+    -------
+    avg_zenith : float
+        The average zenith angle for the event.
+    avg_azimuth : float
+        The average azimuth angle for the event.
+    """
+    zeniths = []
+    azimuths = []
+    num_good_antennas = []
+    for station in event.get_stations():
+        if station.get_parameter(stationParameters.triggered):
```

Comment:
I agree on this. This function so far is just a basic starting point. I am also not sure how to effectively implement this "untriggering". Maybe we can only allow stations with direction somewhat near the LORA estimate? We could also think about whether to use the median instead of the mean.